client/src/components/Homepage-enter/homepage-enter.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
```

Comment:
homepage where user can click the image component and be directed to products page